cookbooks/fb_stunnel/resources/create_certs.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+action :enable do
+  node['fb_stunnel']['config'].each do |section, config|
+    if !config['_create_self_signed_cert'] || !config['cert'] || !config['key']
+      Chef::Log.debug(
+        "fb_stunnel[#{section}]: Not creating cert: missing " +
+        '"cert", "key", or "create_self_signed_cert"',
+      )
+      next
+    end
+    if ::File.exist?(config['cert'])
+      Chef::Log.debug(
+        "fb_stunnel[#{section}]: Not creating cert: it already exists.",
+      )
+    end
+
+    execute "create key/cert for #{section}" do
+      command 'openssl req -new -newkey rsa:4096 -x509 -sha256 -days 365 ' +
```

Comment:
This will fail if the openssl CLI isn't installed. Which may be ok, but you should probably call that our in the README.